nimbus.fml.yaml

```diff
@@ -181,6 +185,9 @@ features:
             EXPIRES_QUICKLY:
               priority: 100
               max-display-count: 1
+          notification-config:
+            polling-interval: 180 # 3 minutes
```

Comment:
Can I clarify why this is required to be 3 minutes? Reason for asking is that typically Workers are restricted by the OS with regard to how often they can be performed